bindings/go/src/fdb/errors.go

```diff
@@ -49,7 +49,9 @@ func (e Error) Error() string {
 // SOMEDAY: these (along with others) should be coming from fdb.options?
 
 var (
-	errNetworkNotSetup = Error{2008}
+	errNetworkNotSetup          = Error{2008}
+	errNetworkAlreadySetup      = Error{2009}
+	errNetworkCannotBeRestarted = Error{2025}
```

Comment:
These are really not used in this PR; I could remove them, I thought to add them and leave them here for completeness.